ai_tools/elasticgraph-mcp-server/pyproject.toml

```diff
@@ -40,10 +40,10 @@ packages = ["src/elasticgraph_mcp"]
 [dependency-groups]
 dev = [
     "ruff~=0.14.0",
-    "pytest~=8.4.1",
+    "pytest~=9.0.0",
```

Comment:
@BrianSigafoos-SQ IIRC the tests for `elasticgraph-mcp-server` are not run on CI so it's possible that there's a breaking change here that impacts us.

Can you try this out and report back if this is safe to merge?

And also, is there anything we should add to the CI build for the MCP server?